Documentation/ceph-disaster-recovery.md

```diff
@@ -229,6 +229,166 @@ kubectl -n rook-ceph scale deployment rook-ceph-operator --replicas=1
 
 The operator will automatically add more mons to increase the quorum size again, depending on the `mon.count`.
 
+
+## Recovering Mon Quorum using OSDs
+
+If all MONs fail, we can recover monitor store from OSDs. The steps in this section are based on the following document.
+
+https://docs.ceph.com/en/latest/rados/troubleshooting/troubleshooting-mon/#recovery-using-osds
+
+Known limitations are written in the following document.
+
+https://docs.ceph.com/en/latest/rados/troubleshooting/troubleshooting-mon/#known-limitations
+
+### Prerequisites
+
+1. OSDs are not corrupted.
+2. The rook [toolbox](./ceph-toolbox.md) is deployed in the target cluster's namespace.
+
+### Set variables related to the environment
+
+```console
+FSID="<The value after --fsid is written in the mon deployment manifest.>"
```

Comment:
This is a lot of scripting in documentation. What if we provide this as one or more bash scripts in the `test/scripts` folder? The documentation then can focus on describing the steps, and the details can be left in the script.

Reply:
+1